module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,19 @@
+#include "sort.hpp"
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> listToSort) {
+    auto sortPred = [](const std::string& lhs, const std::string& rhs) {
+        if (lhs.size() == rhs.size()) {
+            return lhs <= rhs;
+        }
+
+        return lhs.size() <= rhs.size();
+    };
+

```

Comment:
Just pass this predicate to sort. Don't create temporary variable to hold this predicate.